config.go

```diff
@@ -77,7 +81,7 @@ func WithEmptyMessagesLimit(limit uint) ClientOption {
 	}
 }
 
-func WithBetaVersion(betaVersion string) ClientOption {
+func WithBetaVersion(betaVersion BetaVersion) ClientOption {
```

Comment:
I have a suggestion. According to the Claude documentation: anthropic-beta: If passing in multiple, use a comma separated list without spaces, e.g. beta1,beta2. Here, the parameters can be restructured as: func withBetaVersion(versions ...BetaVersion) requestSetter.

Reply:
Sounds good to me! I'll make that change now.

Reply:
ready for another look